src/Tribe/Views/V2/Views/Latest_Past_View.php

```diff
@@ -12,7 +12,7 @@
 use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
 use Tribe__Context;
 
-class Recent_Past_View extends View {
+class Latest_Past_View extends View {
 
 	use List_Behavior;
 	/**
```

Comment:
:no_entry_sign: Missing doc comment for class Latest_Past_View

<sub>hash: bfd008390e43cb26f971dca0ca5d273c</sub>